src/recorder/creaperproject.cpp

```diff
@@ -62,19 +62,19 @@ CReaperItem::CReaperItem ( const QString& name, const STrackItem& trackItem, con
 
     QTextStream sOut ( &out );
 
```

Comment:
All of the recorder files _want_ to flush eash line.

Reply:
Actually, in that case, could we drop the `; sOut` from most of the lines and just have one big statement...
```
sOut << "    <ITEM " << '\n'
     << "      FADEIN 0 0 0 0 0 0" << '\n'
     ...
     << "      >" << endl;

sOut.flush();
```
(I'd still prefer the double flush.)

Reply:
> Actually, in that case, could we drop the ; sOut from most of the lines and just have one big statement...

Done (here and in line 118; the other places with 3 lines or less did not make sense, especially since `clang-format` would start putting those into a single line then).

> (I'd still prefer the double flush.)

Do you mean `endl`? If so, why, is it any safer that way?
What should I do -- add a `#define` as initially thought? It sounded overkill and looked not that readable to me.

Reply:
It was probably something I read on StackOverflow and may have been about Qt4 (as that's what we were targetting when this code was written).  So it may not apply now.  I think the concern was that sometimes there would be an intermediate buffer, so the first flush (endl) flushed to the file (which was buffered) and the second flush said "Oh, the first buffer is clean, I better flush the file buffer to disk".  Or something.

It's fairly important to avoid missing the opportunity to save what's being written here -- the audio has been saved but the track timings relative to each other aren't easy to recover (but I've a script which helps).